package.json

```diff
@@ -4,9 +4,11 @@
   "scripts": {
     "proptypes": "cross-env BABEL_ENV=test babel-node --extensions \".tsx,.ts,.js\" ./scripts/generateProptypes.ts",
     "deduplicate": "node scripts/deduplicate.js",
-    "argos": "node ./scripts/pushArgos.js",
     "benchmark:browser": "yarn workspace benchmark browser",
+    "build:release": "lerna run --parallel --scope \"@material-ui/*\" build",
     "build:codesandbox": "lerna run --parallel --scope \"@material-ui/*\" build",
+    "release": "yarn build:release && lerna publish --no-changelog --dist-tag next --contents build",
+    "publish:dry-run": "lerna publish --no-changelog --dist-tag next --contents build --no-git-tag-version --no-push --registry=\"http://localhost:4873/\"",
```

Comment:
Could you add some docs to the release process while we're at it? It's unclear how this should be used.

Reply:
Sure, I would be happy too. The objective of the change was to test that the release process works end-to-end. I have used https://github.com/verdaccio/verdaccio.

Reply:
Done https://trello.com/c/H7Tz29ft/2763-release-steps

Reply:
I meant documenting as in adding a section to CONTRIBUTING.md so that we can always keep it in sync.

Reply:
I think that moving it to git would have the opposite effect, it will make updating the instructions more painful (opening a PR vs. doing the edits directly), so less likely to happen.

Reply:
> (opening a PR vs. doing the edits directly),

I suspected that you're less and less interested in having your work review. But you seem to be doing fine with just committing if you don't want people to review it. Why would that not work here?

Reply:
> it will make updating the instructions more painful 

Then we need to work on the pain points when opening a PR. What are frequent problems you run into?

Reply:
I think that keeping in sync and helping with reviews are two different topics. I thought the concern was with keeping in sync with the code.

I had assumed that we wouldn't need to review changes to the release workflow instructions, that they are meant to be an unbiased list of steps to follow, that if there is a mistake in it, anybody would go and update the instructions to fix them. 

I had also assumed that reviewing the release workflow would require a different discussion. I'm *still* happy to do it if you think it's important. React has it documented in git: https://github.com/facebook/react/blob/master/scripts/release/README.md.

Reply:
> I had assumed that we wouldn't need to review changes to the release workflow instructions, that they are meant to be an unbiased list of steps to follow, that if there is a mistake in it, anybody would go and update the instructions to fix them.

Same exact reasons why we keep any documentation checked-in. The release workflow isn't something special that needs to be kept in a separate, inaccessible place. At least I can't see a reasons from what you've told. Maybe there are other factors you're not sharing but I can't address those.

Reply:
I have moved the content of the Trello card to a readme.